client/views/session.tpl

```diff
@@ -0,0 +1,15 @@
+<p class="clearfix">
+{{#if current}}
+	<strong>{{agent}}</strong>
+	<a href="https://ipinfo.io/{{ip}}" target="_blank" rel="noopener">{{ip}}</a>
```

Comment:
Nitpick but any specific reason why you used ipinfo.io over geoiptool.com I used in #986? I liked that it shows a more precise location on the map + zoomable. ipinfo.io isn't doing too well (though I know this is not a perfect metric, might as well display it in a useful way).

Reply:
I have used ipinfo.io before, I think it's a nicer service overall.

geoiptool.com reports US for my IPv6 address, so it's that good.

Reply:
Fair enough :+1: